.sail/Dockerfile

```
@@ -1,14 +1,19 @@
 FROM codercom/ubuntu-dev-go:latest
+SHELL ["/bin/bash", "-c"]
+RUN sudo apt-get update && \
+  sudo apt-get install -y htop && \
```

Trailing `&& \`, will squash and merge afterwards
```suggestion
  sudo apt-get install -y htop
```